specs/commandline/inspect.md

```diff
@@ -0,0 +1,205 @@
+      ├──certificates:
+      		├──SHA1 fingerprint: 2f1rr5b8455381frdajc83b4bd305b743cc9513u
+      ├──payload: //descriptor of the target artifact manifest that is being signed.
+      		├──mediaType: application/vnd.oci.image.manifest.v1+json
+        	├──digest: sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a3c333
+        	├──size: 16724
+```
+
+## Usage signatures on an OCI artifact identified by a tag
+
+A tag is resolved to a digest first before inspect.
+
+```shell
+# Prerequisites: Signatures are stored in a registry referencing the signed OCI artifact
+
+notation inspect localhost:5000/net-monitor:v1
+```
+An example output:
+Resolved artifact tag `v1` to digest `sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9` before inspect.
+Warning: The resolved digest may not point to the same signed artifact, since tags are mutable.
+
+```shell
+localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+├──application/vnd.cncf.notary.signature
+    ├── sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+      ├──signed attributes:
+    		├──contentType: application/vnd.cncf.notary.payload.v1+json
+        	├──signingScheme: notary.default.x509
+        	├──signingTime: 2022-04-06T07:01:20Z
+        	├──expiry: 2022-10-06T07:01:20Z
+        	├──verification plugin: com.example.nv2plugin     //extended attributes used by Notary v2 to support plugins
+      ├──user-defined attributes:
+      		├──io.wabbit-networks.buildId: 123    //Notary v2 payload annotations is shown here has user defined metadata.
+      ├──unsigned attributes:
+      		├──io.cncf.notary.timestampSignature: <Base64(TimeStampToken)>    //how TSA response (time stamp token) is represented in this header.
+        	├──io.cncf.notary.signingAgent: notation/1.0.0    //provides the identifier of a client (e.g. Notation) that produced the signature
+      ├──certificates:
+      		├──SHA1 fingerprint: 2f1cc5b8455381cdefac83b4bd305b789cc9c16e
+     	├──payload: //descriptor of the target artifact manifest that is being signed.
+ 		      ├──mediaType: application/vnd.oci.image.manifest.v1+json
+        	├──digest: sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a3c333
+        	├──size: 16724
+      
+    ├── sha256:bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
+      ├──signed attributes:
+      		├──contentType: application/vnd.cncf.notary.payload.v1+json
+        	├──signingScheme: notary.signingAuthority.x509
+        	├──authenticSigningTime: 2022-04-06T07:01:20Z
+        	├──expiry: 2022-10-06T07:01:20Z
+        	├──verification plugin: com.example.nv2plugin     //extended attributes used by Notary v2 to support plugins.
+      ├──unsigned attributes:
+      		├──io.cncf.notary.timestampSignature: <Base64(TimeStampToken)>      //how TSA response (time stamp token) is represented in this header.
```

Comment:
Please remove comments since they not part of output.
`//how TSA response (time stamp token) is represented in this header.`